src/pieces/gui.py

```diff
@@ -76,28 +76,31 @@ def print_instructions():
 def print_help():
     print()
     print("Available Commands:")
-    print("  run          - Starts a looped version of the CLI that only requires you to type the flag")
+    print("  run             - Starts a looped version of the CLI that only requires you to type the flag")
     print()
-    print("  list         - List all assets")
-    print("  list x       - Lists maximum of x number of assets")
-    print("  list apps    - List all registered applications")
-    print("  list models  - List all registered AI models")
+    print("  list            - List all assets")
+    print("  list x          - Lists maximum of x number of assets")
+    print("  list apps       - List all registered applications")
+    print("  list models     - List all registered AI models")
     print()
-    print("  open         - Opens your current or most recent asset")
-    print("  open x       - Open an asset in a list. The CLI must be running")
-    print("  save         - Save the current asset (not currently functional)")
-    print("  delete       - Deletes the current or most recent asset.")
-    print("  create       - Creates a new asset based on what you've copied to your clipboard")
+    print("  open            - Opens your current or most recent asset")
+    print("  open x          - Open an asset in a list. The CLI must be running")
+    print("  save            - Save the current asset (not currently functional)")
+    print("  delete          - Deletes the current or most recent asset.")
+    print("  create          - Creates a new asset based on what you've copied to your clipboard")
     print()
-    print("  ask \"ask\"    - Asks a single question to GPT 3.5 for now. Default timeout set to 10 seconds")
+    print("  change_model    - Lists all the avaiable models that are on your machine")
+    print("  change_model x  - Change the model that is used in the ask command defaults to chatGPT 3.5 similar to list models")
```

Comment:
small reminder when we resolve the readme discussion on the commands that we also update these 2 here as well + their descriptions